spec/generators/install_generator_spec.rb

```diff
@@ -52,6 +53,20 @@
 
       expect(manifest).not_to contain("delayed/backend/active_record/jobs")
     end
+
+    it "skips models that aren't backed by the database" do
+      begin
+        class ModelWithoutDBTable < ActiveRecord::Base; end
+        stub_generator_dependencies
+        manifest = file("app/dashboards/dashboard_manifest.rb")
+
+        run_generator
+
+        expect(manifest).not_to contain("model_without_db_table")
+      ensure
+        remove_constants :ModelWithoutDBTable
```

Comment:
@JoelQ I want to make sure that we always clean up our constants, even if the test fails. It probably wouldn't affect much in this case, because we're not using `ModelWithoutDBTable` anywhere else. But in the past I've gotten some failures that were hard to track down because we weren't correctly cleaning up like this.

Also, in case you were asking about the ruby syntax side of things, yeah - you can't just use `ensure` or `rescue` inside a normal Ruby block. I believe those only work in methods.